kolibri/core/assets/src/views/AppBar.vue

```diff
@@ -39,79 +39,7 @@
             <slot name="totalPointsMenuItem"></slot>
           </div>
 
-          <UiButton
-            ref="userMenuButton"
-            type="primary"
-            color="clear"
-            class="user-menu-button"
-            :class="$computedClass({ ':focus': $coreOutline })"
-            :ariaLabel="$tr('userMenu')"
-            @click="handleUserMenuButtonClick"
-          >
-            <template #icon>
-              <KIcon
-                icon="person"
-                :style="{ fill: $themeTokens.textInverted, height: '24px', width: '24px', top: 0, }"
-              />
-            </template>
-            <span v-if="isUserLoggedIn" class="username" tabindex="-1">{{ dropdownName }}</span>
-            <KIcon
-              icon="dropdown"
-              :style="{ fill: $themeTokens.textInverted, height: '24px', width: '24px', top: 0, }"
-            />
-          </UiButton>
-
-          <CoreMenu
-            v-show="userMenuDropdownIsOpen"
-            ref="userMenuDropdown"
-            class="user-menu-dropdown"
-            :isOpen="userMenuDropdownIsOpen"
-            :raised="true"
-            :containFocus="true"
-            :showActive="false"
-            :style="{ backgroundColor: $themeTokens.surface }"
-            @close="handleCoreMenuClose"
-            @shouldFocusFirstEl="findFirstEl()"
-          >
-            <template v-if="isUserLoggedIn" #header>
-              <div class="role">
-                {{ coreString('userTypeLabel') }}
-              </div>
-              <div>
-                <UserTypeDisplay
-                  :distinguishCoachTypes="false"
-                  :userType="getUserKind"
-                />
-              </div>
-              <div v-if="isSubsetOfUsersDevice && userIsLearner" data-test="syncStatusInDropdown">
-                <div class="sync-status">
-                  {{ $tr('deviceStatus') }}
-                </div>
-                <SyncStatusDisplay
-                  :syncStatus="mapSyncStatusOptionToLearner"
-                  displaySize="large"
-                />
-              </div>
-            </template>
-
-            <template #options>
-              <component :is="component" v-for="component in menuOptions" :key="component.name" />
-              <CoreMenuOption
-                :label="$tr('languageSwitchMenuOption')"
-                icon="language"
-                style="cursor: pointer;"
-                @select="handleChangeLanguage"
-              />
-              <LogoutSideNavEntry v-if="isUserLoggedIn" />
-            </template>
-
-            <template #footer>
-              <!-- Only show this when on a SoUD -->
-              <div v-if="showSoudNotice" class="role" data-test="learnOnlyNotice">
-                <LearnOnlyDeviceNotice />
-              </div>
-            </template>
-          </CoreMenu>
+          <span v-if="isUserLoggedIn" class="username" tabindex="-1">{{ dropdownName }}</span>
```

Comment:
Since it isn't a dropdown anymore, maybe `dropdown` name should be renamed to avoid future confusion?